CONTRIBUTING.md

```diff
@@ -84,6 +84,39 @@ About two tasks are one time for the life time. You can continue to use the same
      - When the PR has 'Approved' flag from corresponding maintainer.
    - If you feel there is delay, feel free to add a comment, discuss the same in Slack channel, or send email.
 
+## Becoming member of gluster.org team in github
+
+This section details the process of becoming part of gluster.org team.
+The member list is reviewed by the current maintainers along with infra team time to time.
+
+### How to create a presence/identity in the Gluster.org projects
+
+Words 'MUST', 'SHOULD' etc to be referred from [RFC2119](https://tools.ietf.org/html/rfc2119)
+
+* MUST have enabled [two-factor authentication](https://help.github.com/articles/about-two-factor-authentication) on their GitHub account
+* SHOULD have made multiple contributions to the project or community. Contribution may include, but is not limited to:
+  - Authoring or reviewing PRs on GitHub
+  - Filing or commenting on issues on GitHub
+  - Contributing to subproject, or community discussions (e.g. meetings, Slack, email discussion forums, Stack Overflow)
+* MUST be subscribed to [gluster-users](https://lists.gluster.org/mailman/listinfo/gluster-users)/[gluster-devel](https://lists.gluster.org/mailman/listinfo/gluster-devel) mailing lists
+* MUST have read the [contributor guide](.)
+* SHOULD Actively contributing to 1 or more subprojects for 30 days.
```

Comment:
SHOULD be actively contributing to 1 or more sub-projects for 30 days

Reply:
Done.